solr/webapp/web/index.html

```diff
@@ -53,8 +53,8 @@
   <script src="libs/jquery-2.1.3.min.js"></script>
   <script src="libs/chosen.jquery.js"></script>
   <script src="libs/jquery.jstree.js"></script>
-  <script src="libs/angular.js"></script>
-  <script src="libs/angular-chosen.js"></script>
+  <script src="libs/angular.min.js"></script>
```

Comment:
my diff in the comment didn't include this, but yeah this is of course necessary too.   there's some other references that would need updating too?

Reply:
no other references